src/main/io/serial_cli.c

```diff
@@ -3926,11 +3955,16 @@ static void cliResource(char *cmdline)
                 pin = atoi(pch);
                 if (pin < 16) {
                     ioRec_t *rec = IO_Rec(IOGetByTag(DEFIO_TAG_MAKE(port, pin)));
-                    if (rec) {
+                    if (rec) { 
                         *tag = DEFIO_TAG_MAKE(port, pin);
-                        cliPrintf("Resource is set to %c%02d!", port + 'A', pin);
+#ifndef CLI_MINIMAL_VERBOSITY
+                        cliPrintf("Resource is set to %c%02d!\r\n", port + 'A', pin);
+                        resourceCheck(resourceIndex, index, DEFIO_TAG_MAKE(port, pin));
```

Comment:
I like the warning. Would it make sense to additionally check for illegal double assignments (like multiple assignments of the same resource type to the same pin), and throw an error on these?